### ISSUE TITLE
xqemu quietly quits to prompt immediately after launch, regardless of options set

### ISSUE THREAD

**Issue by Lethaltail** [espes/xqemu#72]
I've used multiple computers and multiple versions of Windows, the same thing happens. I get no information of why I return to the MSYS prompt.

**Comment by JayFoxRox** [comment]
Lack of information: What command line did you use to compile, run, what methods did you use to dump your roms (remember: don't discuss dodgy shit), did you confirm their validity? What version of windows, what version of the compiler? ...

**Comment by Lethaltail** [comment]
I followed the Windows build instructions on Windows 7, as well as multiple versions of Windows 10, on multiple computers. The compiler was downloaded along with everything else the instructions asked for about two days ago. Everything was done under MSYS2 x86_64. I've found very little info on dumping the ROM, and *<we don't talk about the rest of this paragraph>*


The emulator reacts the same way regardless of if I set no command line options, or everything suggested on the wiki. I'm put back at the MSYS2 prompt as soon as I start the emulator, no messages from it.
